CNV_pipeline/wdl_scripts/CNV_Somatic_Pair_Workflow.wdl

```diff
@@ -4,8 +4,8 @@
 #
```

Comment:
Just trying to understand the diffs in this file. Are these necessary for consolidating WDLs?

Reply:
I think it comes from me updating the file to its latest version on terra. I don't remember changing this file